spp_change_request/models/mixins/source_mixin.py

```diff
@@ -741,3 +741,25 @@ def action_attach_documents(self):
     #         raise UserError(
     #             _("There are no directories defined for this change request.")
     #         )
+
+    def open_registrant_details_form(self, name="Group Details"):
+        self.ensure_one()
+        res_id = self.registrant_id.id
+        form_id = self.env.ref("g2p_registry_group.view_groups_form").id
+        action = self.env["res.partner"].get_formview_action()
+        context = {
+            "create": False,
+            "edit": False,
+            "hide_from_cr": 1,
+        }
+        action.update(
+            {
+                "name": _(name),
```

Comment:
I don't think this is going to work for translation. You should translate the string at the source where it is declared.

Reply:
Hi Jeremi, i moved this function to change_request_activate_deactivate_card.py and addressed the issue you stated here. Thanks.